supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalBackend.scala

```diff
@@ -1,29 +1,227 @@
+import cromwell.backend.BackendLifecycleActor.{BackendJobExecutionAbortFailedResponse, BackendJobExecutionAbortSucceededResponse, JobAbortResponse}
 import cromwell.backend._
-import wdl4s.Call
+import cromwell.core.CallContext
+import org.slf4j.LoggerFactory
+import wdl4s._
+import wdl4s.expression.ValueEvaluator
+import wdl4s.values.{WdlFile, WdlValue}
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
+import scala.language.postfixOps
+import scala.util.{Failure, Success, Try}
 
-class LocalBackend extends BackendJobExecutionActor {
-  /**
-    * Restart or resume a previously-started job.
-    */
-  override def recover: Future[BackendJobExecutionResponse] = ???
+object LocalBackend {
+  val logger = LoggerFactory.getLogger("LocalBackend")
+  // TODO Support GCS ?
+  val fileSystems = List(FileSystems.getDefault)
+
+  private def splitFqn(fullyQualifiedName: FullyQualifiedName): (String, String) = {
+    val lastIndex = fullyQualifiedName.lastIndexOf(".")
+    (fullyQualifiedName.substring(0, lastIndex), fullyQualifiedName.substring(lastIndex + 1))
+  }
+
+  case class Command(argv: Seq[String]) {
+    override def toString = argv.map(s => "\"" + s + "\"").mkString(" ")
+  }
+}
+
+class LocalBackend(override val jobDescriptor: BackendJobDescriptor,
+                   override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendJobExecutionActor with SharedFileSystem {
+
+  import LocalBackend._
+  import better.files._
+  import cromwell.core.PathFactory._
+
+  import scala.sys.process._
+
+  // Mutable process variable assigned when the execute method is called.
+  // Needs to be accessible to be killed when the job is aborted.
+  private var process: Option[Process] = None
+
+  val workflowDescriptor = jobDescriptor.descriptor
+  val jobPaths = new JobPaths(workflowDescriptor, backendConfiguration, jobDescriptor.key)
+  val fileSystemsConfig = backendConfiguration.getConfig("filesystems")
+  override val sharedFsConfig = fileSystemsConfig.getConfig("local")
+
+  val call = jobDescriptor.key.call
+  val callEngineFunction = {
+    val callContext = new CallContext(
+      jobPaths.callRoot.toString,
+      jobPaths.stdout.toAbsolutePath.toString,
+      jobPaths.stderr.toAbsolutePath.toString
+    )
+
+    new LocalCallEngineFunctions(fileSystems, callContext)
+  }
+
+  val evaluator = {
+    val declarations = workflowDescriptor.workflowNamespace.workflow.declarations ++ call.task.declarations
+    val unqualifiedWorkflowInputs = workflowDescriptor.inputs map {
+      case (fqn, v) => splitFqn(fqn)._2 -> v
+    }
+    val knownInputs = unqualifiedWorkflowInputs ++ jobDescriptor.symbolMap
+    val lookup = WdlExpression.standardLookupFunction(knownInputs, declarations, callEngineFunction)
+    new ValueEvaluator(lookup, callEngineFunction)
+  }
```

Comment:
I'm just offering some alternatives here...

I was surprised to see that you're creating a `ValueEvaluator`.  What about instead of `val evaluator`, it was `val lookup` then the `evaluate()` method a few lines below can be:

```scala
private def evaluate(wdlExpression: WdlExpression) = wdlExpression.evaluate(lookup, callEngineFunctions)
```